hue/hue/base/params.yaml

```diff
@@ -0,0 +1,12 @@
+---
+varReference:
+  - path: stringData/hue.ini
+    kind: Secret
+  - path: stringData/hive-site.xml
+    kind: Secret
+  - path: stringData/DATA_SOURCE_NAME
+    kind: Secret
+  - path: metadata/annotations/volume.beta.kubernetes.io/storage-class
```

Comment:
This does not template the variable correctly..see https://github.com/opendatahub-io/odh-manifests/blob/master/jupyterhub/jupyterhub/base/params.yaml#L7